static/src/js/actions/index.js

```diff
@@ -8,16 +8,30 @@ import {
 } from './search'
 import { changeMap } from './map'
 import { changeUrl } from './urlQuery'
+import {
+  changeCmrFacet,
+  changeFeatureFacet
+} from './facets'
+import {
+  masterOverlayPanelResize,
+  masterOverlayPanelDragStart,
+  masterOverlayPanelDragEnd
+} from './ui'
 
 const actions = {
   getCollections,
   getGranules,
   changeQuery,
   changeMap,
   changeFocusedCollection,
-  clearFilters,
   changeUrl,
-  searchNlp
+  clearFilters,
+  searchNlp,
```

Comment:
Alpha order? Or are we grouping these by something?

Reply:
I alpha ordered all the things. Just to make _you_ happy, Ryan :)